### PR TITLE
refine linux router guide

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

File: docusaurus/docs/guides/deployments/10-linux/20-router/50-migrate.mdx [hunk 1]
```diff
@@ -3,48 +3,38 @@ title: Migrate a Router Installation
...
+1. Ensure service is disabled and state is clean.
+    
+    ```text
+    sudo systemctl disable --now ziti-router.service
+    sudo systemctl clean --what=state ziti-router.service
+    ```
 
-```bash
-#!/bin/bash
+1. Duplicate the router-related parts of the configuration to the service's working directory.
```
Comment (dovholuknf) [inline docusaurus/docs/guides/deployments/10-linux/20-router/50-migrate.mdx:15]: though Duplicate is a synonym for copy, "copy" is a much more natural sounding word here

File: docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ identity:
 
 ### Private Key
 
-The shape of the [`identity`](/reference/30-configuration/conventions.md#identity) section is conventional for routers and controllers, but the `key` sub-property behaves differently for routers. If a private key exists in the path, then it will be used to sign router enrollment certificate requests. If it does not exist, then a private key will be locally generated and stored in that path.
+The [`identity`](/reference/30-configuration/conventions.md#identity) configuration section is conventional for routers and controllers, but the `key` sub-property behaves differently for routers. If a private key exists in the path, then it will be used to sign router enrollment certificate requests. If it does not exist, then a private key will be locally generated and stored in that path.
```
Comment (dovholuknf) [inline docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md:31]: the changes above through here all still reference-esque and not "guide" esque imo.

Comment (dovholuknf) [reply docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md:31]: > configuration section is conventional for routers and controllers

"conventional" is too formal and not conversational and sounds/reads awkwardly to me. it should be replaced with "common" or anything less clinical imo

**Review by dovholuknf — APPROVED** [review]

File: docusaurus/docs/guides/deployments/10-linux/20-router/50-migrate.mdx (hunk 1 above)
Comment (qrkourier) [reply docusaurus/docs/guides/deployments/10-linux/20-router/50-migrate.mdx:15]: Improved in https://github.com/openziti/ziti-doc/pull/868

File: docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md (hunk 2 above)
Comment (qrkourier) [reply docusaurus/docs/guides/deployments/10-linux/20-router/30-configuration.md:31]: Improved in https://github.com/openziti/ziti-doc/pull/868

**Pull request merged**